lib/web_ui/test/path_test.dart

```diff
@@ -252,4 +254,18 @@ void main() {
     // Expect original path to stay the same.
     expect(path.getBounds(), const Rect.fromLTRB(25, 30, 100, 200));
   });
+
+  // Regression test for https://github.com/flutter/flutter/issues/44470
+  test('Should handle contains for devicepixelratio != 1.0', () {
+    js_util.setProperty(html.window, 'devicePixelRatio', 4.0);
+    window.debugOverrideDevicePixelRatio(4.0);
+    final path = Path()
+      ..moveTo(50, 0)
+      ..lineTo(100, 100)
+      ..lineTo(0, 100)
+      ..lineTo(50, 0)
+      ..close();
+    expect(path.contains(Offset(50, 50)), isTrue);
+    js_util.setProperty(html.window, 'devicePixelRatio', 1.0);
```

Comment:
also set the `debugOverrideDevicePixelRatio` back to `1.0`?

Reply:
Done.